vnext/PropertySheets/React.Cpp.props

```diff
@@ -3,14 +3,14 @@
 
   <PropertyGroup Label="Globals">
     <DefaultLanguage>en-US</DefaultLanguage>
-    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
     <!-- Use 19H1 SDK (10.0.18362.0)  Support running on RS3+ (10.0.16299.0) -->
     <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>$(DefaultPlatformToolset)</PlatformToolset>
```

Comment:
Please leave `v142` instead.
Whenever a new VS version or `v143`, etc. gets released, we want to use a specific compiler collection to avoid unexpected incompatibilities.

Reply:
We control that with the workflow build parameters. That way we can lock the requirements for a stable build branch but let master keep us informed as versions update. So we catch breaks sooner.